### ISSUE TITLE
create initial react setup to a frontend folder

### ISSUE THREAD

**Issue by hall500** [hall500/transpaynet-frontend#1]
Create a PR for the initial setup of a starter react template.

**Comment by ashfaqhaq** [comment]
so just a basic create react app for a pull request?

Sure!

**Comment by hall500** [comment]
Yeah, this is supposed to lead to a series of components which will manage the transpaynet API
